tools/mqppep/mqppep_preproc.xml

```diff
@@ -0,0 +1,531 @@
+<tool
+  id="mqppep_preproc"
+  name="MaxQuant Phosphopeptide Preprocessing"
+  version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@"
+  python_template_version="3.5"
+  profile="21.05"
+  >
+    <description>
+        Transform the output of MaxQuant for phosphoproteome-enriched samples to prepare it for statistical anlaysis.
+    </description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements"/>
+    <command detect_errors="exit_code"><![CDATA[
+      (
```

Comment:
I don't think you need to use the brackets here

Reply:
I will try ...